Traceback (most recent call last):
  File "C:\Python312\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Python312\Lib\site-packages\nbclient\client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\jupyter_core\utils\__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\asyncio\base_events.py", line 664, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\nbclient\client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "C:\Python312\Lib\site-packages\nbclient\client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Python312\Lib\site-packages\nbclient\client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# rekent de oppervlakte van een rechthoek uit 
# hoogte: de hoogte van de rechthoek
# breedte: de breedte van de rechthoek
# return: de oppervlakte van de rechthoek

def Oppervlakte(hoogte, breedte):
	Opp = hoogte * breedte
	return Opp
	
Opp_rechthoek = Oppervlakte(2,3)

print(f"Oppervlakte: {Opp_rechthoek})
------------------


[1;36m  Cell [1;32mIn[1], line 12[1;36m[0m
[1;33m    print(f"Oppervlakte: {Opp_rechthoek})[0m
[1;37m          ^[0m
[1;31mSyntaxError[0m[1;31m:[0m unterminated f-string literal (detected at line 12)


